srcpkgs/vile/template

```diff
@@ -11,7 +11,7 @@ maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-only"
 homepage="http://invisible-island.net/vile/vile.html"
 distfiles="https://invisible-mirror.net/archives/vile/current/vile-${version}.tgz"
-checksum=78253ec3f7ae5f4f9d4799a3c8bc35b85b47d456f2ac172810008a48e4609815
+checksum=8fe0dfa60179d4b7dd2750f116cd4396d4cd3e07d8a54d142a36c84f4a82feef
 
 post_extract() {
 	sed -i 's/\[\[//g' filters/sh-filt.l
```

Comment:
```suggestion
	vsed -e 's/\[\[//g' -i filters/sh-filt.l
```
and the same in post_configure

Reply:
sed in post_configure should be changed to vsed as well

Reply:
Sorry, I forgot.
Changed in the new commit